### ISSUE TITLE
Fix Playbook Accessibility Violations

### ISSUE THREAD

**Issue by JillTunick** [GSA/playbooks-wdsv2#493]
Make fixes to remediate accessibility violations discovered during manual testing of the AMP Apr 22 Playbooks Prod report (https://gsa.levelaccess.us/public/reporting/view_report.php?report_id=267067#)
